docs/index.rst

```diff
@@ -36,20 +65,107 @@ The plugin may be used within a north *task* or *service*. Both of these are cre
         .. code-block:: JSON
 
             {
-                "sensor": {
-                    "temperature": {
-                        "node": "ns=1;i=1013",
+                "sinusoid": {
+                    "sinusoid": {
+                        "node": "ns=3;i=1017",
                         "type": "Float"
                     }
                 }
             }
 
-        - sensor is an asset name
-        - temperature is a datapoint name
+        - sinusoid is an asset name
+        - sinusoid is a datapoint name
 
       - **Source**: The source of the data to be sent, this may be the *readings* or *statistics* data.
 
-  - Click *Next*
+The OPC UA Security tab contains a set of configuration items that is used for setting the security between the client and the OPC UA Server.
+
++------------------------+
+| |opcuaclient_security| |
++------------------------+
+
+  - **Security Mode**: Specify the OPC/UA security mode that will be used to communicate with the OPC/UA server.
```

Comment:
The terms OPC UA, OPC/UA and OPCUA all appear in the document. I suggest OPC UA everywhere.